treeherder/log_parser/tasks.py

```diff
@@ -95,6 +95,48 @@ def parse_job_logs(project, tasks):
         create_taskset(callback_group, callback)
 
 
+def parse_job_log(project, func_name, routing_key, job_guid, job_log_id):
+    """
+    Schedule the log-related tasks to parse an individual log
```

Comment:
I presume this is replacing ``parse_job_logs`` but we're not quite able to remove it at this time?  I don't see it references, but presumably we could have items in the queue already that would call it.  Should we perhaps just put a ``# TODO: remove this when but xxx is fixed`` in the docstring of ``parse_job_logs``?